developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables.md

```diff
@@ -77,8 +81,8 @@ Every deploy folder has a summary job. This job consists of a **summary.txt** fi
 ### Artifact Pushes
 
 1.The default configuration uploads the artifacts (zip, jar, etc files) to Azure blob and the docker containers to the configured container registry.
-2.Docker container push mandatorily requires a hub account. But if you decide to not use Azure storage blob to store artifacts, then you can change the configuration in Jenkins jobs to disable to push to Azure blob.
-3.Go to the Jenkins ArtifactUpload jobs and Deploy jobs and change the order from ArtifactRepo JenkinsJob to JenkinsJob ArtifactRepo.
+2.It is mandatory to have a hub account to push the docker container. But if you decide to not use the Azure storage blob to store artifacts, then you can change the configuration in Jenkins jobs to disable to push to Azure blob.
```

Comment:
@keshavprasadms please clarfify if this sentence " then you can change the configuration in Jenkins jobs to disable to push to Azure blob." means that "you can modify the configuration to disable the Jenkins job so that the docker container is pushed to the Azure blob." If not, please clarify further.

Reply:
@Sunitashelar Rewrote this statement